src/css/base.scss

```diff
@@ -132,3 +163,58 @@ form {
     }
   }
 }
+
+@media(max-width: 500px) {
+  main {
+    display: flex;
+    flex-direction: column;
+  }
+
+  header {
+    height: auto;
+    h2{
+      font-size: 1rem;
+    }
+    .tagline{
+      display: none;
+    }
+    input {
+      width: auto;
+    }
+  }
+
+  form {
+    min-height: 40vh;
+    img {
+      width: 50%;
+    }
+  }
+  .trip-cards{
+    .card-section {
+      @include flexControl(row);
+      height: 95%;
+      overflow: scroll;
+    }
+    .trip-card {
+      min-width: 40vw;
+      max-height: 40vh;
+      h4, h5, h6 {
+        padding: 1vh;
+        margin: 0;
+      }
+      h4 {
+        font-size: 1rem;
+      }
+      h5 {
+        font-size: .8rem;
+      }
+      h6 {
+        font-size: .2rem;
+      }
+    }
+  }
+  .previous-trips {
+    min-height: 20vh;
+  }
+
+}
```

Comment:
500 px seems a bit small what phones does that work on?

Reply:
500 felt like a good cut off for most phones actually, all iphones fall under 500 as well as as most androids and even the pixel XL. 
The only space that might be a little crammed is a small tablet, but even they they would have a rotation features to get a wider vw.